web/pgadmin/browser/server_groups/servers/databases/schemas/collations/static/js/collation.ui.js

```diff
@@ -109,6 +109,52 @@ export default class CollationSchema extends BaseUISchema {
         disabled: obj.disableFields,
         deps: ['locale', 'copy_collation'],
       },
+      {
```

Comment:
Just like localle, LC_COLLATE - these new fields should also be disabled when using "copy from" as they're not applicable. Check the docs - https://www.postgresql.org/docs/current/sql-createcollation.html